### DISCUSSION TITLE
Is there any intentional reason why `if_keyword_loc` is not named `keyword_loc` as in unless_node?

### DISCUSSION THREAD

**Discussion by r7kamura** [ruby/prism#2110] [Q&A] [answered]
I think the `unless` keyword is the opposite of the `if` keyword in Ruby syntax[^1].

On the other hand, there is a slight difference in the naming of the two in prism's AST. While `if_node` has the field `if_keyword_loc`, the `unlless_node` has `keyword_loc` instead of `unless_keyword_loc`.

If the property names were identical, I could successfully treat them as opposite pairs in my application using prism, but I had to write a conditional branch because of this difference.

Is there an intentional reason for this difference?

[^1]: Although strictly speaking, there is a difference in whether or not `eislf` can be used.

**Comment by kddnewton** [comment, marked as answer]
No intentional reason. I plan to go through the names of the fields before I release 1.0 to get them all aligned, and this will be one of the fixes.
